cmake/dependencies.cmake

```diff
@@ -17,44 +17,52 @@ list(APPEND CMAKE_MESSAGE_CONTEXT "dep")
 
 morpheus_utils_initialize_cpm(MORPHEUS_CACHE_DIR)
 
-
+# Show some setup variables (only prints if VERBOSE)
 morpheus_utils_print_config()
 
 # First, load the package_config functions
 include(${CMAKE_CURRENT_LIST_DIR}/package_config/register_api.cmake)
 
 # Load direct physical package dependencies first, so we fail early. Add all dependencies to our export set
-rapids_find_package(Protobuf
-  REQUIRED
+rapids_find_package(Protobuf REQUIRED
   BUILD_EXPORT_SET ${PROJECT_NAME}-core-exports
   INSTALL_EXPORT_SET ${PROJECT_NAME}-core-exports
 )
 
-find_package(CUDAToolkit REQUIRED)
+rapids_find_package(CUDAToolkit REQUIRED
+  BUILD_EXPORT_SET ${PROJECT_NAME}-core-exports
+  INSTALL_EXPORT_SET ${PROJECT_NAME}-core-exports
+)
 
 if(MORPHEUS_BUILD_BENCHMARKS)
   # google benchmark
   # - Expects package to pre-exist in the build environment
   # ================
   rapids_find_package(benchmark REQUIRED
-    GLOBAL_TARGETS      benchmark::benchmark
-    BUILD_EXPORT_SET    ${PROJECT_NAME}-exports
-    INSTALL_EXPORT_SET  ${PROJECT_NAME}-exports
-    FIND_ARGS
-    CONFIG
+    GLOBAL_TARGETS benchmark::benchmark
+    BUILD_EXPORT_SET ${PROJECT_NAME}-core-exports
+    INSTALL_EXPORT_SET ${PROJECT_NAME}-core-exports
+    FIND_ARGS CONFIG
   )
 endif()
 
+# glog
+# ====
+morpheus_utils_configure_glog()
+
+# grpc
+# =========
+morpheus_utils_configure_grpc()
+
```

Comment:
glog and grpc should both be picked up transiently through MRC. we just need to make sure MRC is exporting them correctly.

glog is less important to expose, but we could still do it.

Reply:
Same as above

Reply:
Since we use these directly, we should have them listed in our Cmake files. If MRC stops exporting them and we didnt explicitly import the libraries, our build would break.

Reply:
After discussing this further, this seems to be related to a regression where MRC no longer exports the necessary gRPC pieces to allow Morpheus to build. So while we directly use glog and RxCpp, we dont directly use gRPC. So we should tackle the root cause for why gRPC is no longer exported.